llvm/lib/Target/RISCV/MCTargetDesc/RISCVELFStreamer.cpp

```diff
@@ -53,6 +53,8 @@ void RISCVTargetELFStreamer::emitDirectiveOptionPIC() {}
 void RISCVTargetELFStreamer::emitDirectiveOptionNoPIC() {}
 void RISCVTargetELFStreamer::emitDirectiveOptionRVC() {}
 void RISCVTargetELFStreamer::emitDirectiveOptionNoRVC() {}
+void RISCVTargetELFStreamer::emitDirectiveOptionExact() {}
```

Comment:
Perhaps move this before OptionPush for a relatively alphabetical order.

(In the other place OptionArch seems misplaced)

Reply:
I went through and alphabetised all of these.